### DISCUSSION TITLE
How to add supports for Unix domian socket

### DISCUSSION THREAD

**Discussion by Young-Flash** [uNetworking/uWebSockets#1438] [Q&A] [answered]
The author seems to have no intention of adding this feature. But I do need it, and where do I start if I'm going to do it myself? Is there any reference about how to implement it? 
Any suggestions are appreciated!
#1076 #1364 
@alexhultman @gillsoftab @zdm @alexey-v-paramonov @marcj @lmussier @sirzooro

**Comment by Young-Flash** [comment, marked as answer]
I have this done, see [here](https://github.com/Young-Flash/uWebSockets).

**Comment by Young-Flash** [comment]
I am willing to make a PR if you agree. @alexhultman

**Reply by ghost** [reply]
If it compiles on Windows and has a client as well. The uSockets changes look like they would not compile on Windows.

I would also like a connect function and a benchmark run (there are benchmarks over TCP already - modify them to use unix socket)

**Reply by Young-Flash** [reply]
ok I will give thought on them
